README.md

```diff
@@ -71,6 +71,20 @@ ClusterServiceVersions can be collected into `CatalogSource`s which will allow a
 
 Learn more about the components used by OLM by reading about the [architecture] and [philosophy].
 
+# Overall OLM Chart
+
+![OLM](./olm.png)
+
+From the above chart, we can see that there are 2 steps required for a new operator onboard:
+
+1. Develop your operator and push it into the operator repository. You can choose quay.io as an example repository.
```

Comment:
What is an "operator repository"? Can you link to a doc describing it? I think new users might get confused here.